cm/views/default.json

```diff
@@ -1,7 +1,17 @@
 {
-  "NODE_DOCKER_HOSTNAME": "dappley",
-  "NODE_RPC_PORT": 50050,
+  "NODE": {
+    "RPC_PORT": 50050,
+    "DOCKER_HOSTNAME": "dappley",
+    "DOCKER_NETWORK": "go-dappley_default"
+  },
+  "AUTH_SERVER": {
+    "PORT": 8082,
+    "DOCKER_HOSTNAME": "auth-server",
+    "SECRET": "",
+    "DEFAULT_USER": "admin",
+    "DEFAULT_PASS": ""
```

Comment:
Apparently, password `""` does not work. When I login without typing anything in password input box, the input box turns red. I can see the request being sent out and a log message on auth-server saying `[main.ts:45] [error]: at least one of the supplied arguments are invalid { username: 'admin' }`

edit: 
This is probably why: https://github.com/dappley/dappley-stats/pull/15/files#diff-1b325f11a69c1b5eb01e98983d27df5aR143
Maybe just use another default password. I think an empty password should not be allowed.

Reply:
`""` was a conscious decision I made so that I didn't need to put a sample password/secret on github. I assumed that an admin would change both the `secret` & `default_pass`. Hence,
```    
...
"DEFAULT_USER"     : "admin"    // any string with length > 0
"DEFAULT_PASS"     : ""     // any string with length > 0
...
```
in the README. I'd prefer not to add a sample to the default config, I could however stress that the values be changed.

Reply:
Thanks for the clarification. I overlooked the readme.